travis/before_deploy.sh

```diff
@@ -18,21 +18,57 @@
 set -e
 
 cd ${TRAVIS_BUILD_DIR}
+# Setup Git configuration
+COMMIT_USER=$(git log --format='%an' ${TRAVIS_COMMIT}^\!)
+COMMIT_EMAIL=$(git log --format='%ae' ${TRAVIS_COMMIT}^\!)
+git config user.name "Travis CI (for ${COMMIT_USER})"
+git config user.email "${COMMIT_EMAIL}"
```

Comment:
I'm not sure if this is the best way to handle this. My thought was that since we're adding a commit on behalf of another person, we would set the commit to their email address and name.

But perhaps we should have a separate Travis user?